cloudformation/app/app.yaml

```diff
@@ -250,39 +201,6 @@ Resources:
         - Name: HealthCheckId
           Value: !Ref ExternalHealthCheck
 
```

Comment:
Bummer.  We lose health checks since we aren't creating the domain name.  I think we might run into weird non-uniqueness issues if we try to do it.  ¯\_(ツ)_/¯.

Might take a separate stab at this.

Reply:
yeah agree... shouldn't be too hard i guess if you name it w/ the owner ... if tehre are 3 rigs that does mean you'd get 3 alarms for the 1 "prod" url i guess... but woudl start alarming right away so maybe a good push to remind you to set up the prod dns if you haven't arleady

Reply:
Yeah.  I think it will take some experimentation.